catalog/spatial/registry/registry-identification-plugin/src/main/java/org/codice/ddf/registry/identification/IdentificationPlugin.java

```diff
@@ -299,7 +291,8 @@ public void init()
                 .map(Result::getMetacard)
                 .collect(Collectors.toList());
         registryIds.addAll(registryMetacards.stream()
-                .map(this::getRegistryId)
+                .filter(RegistryUtility::hasRegistryId)
+                .map(RegistryUtility::getRegistryId)
```

Comment:
It wasn't checking for registry tag before. Can you just map RegistryUtility::getRegistryId and then filter out null?